public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -182,16 +192,37 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       row.viewSynonyms = true;
     }
   }
-  public selectGenderGraph(g) {
-    if (g === 'Gender Identity') {
-      this.showGenderIdentity = true;
-      this.showGender = false;
+  public selectGraph(g) {
+    this.showGenderGraph = false;
+    this.showAgeGraph = false;
+    this.showSourcesGraph = false;
+    if (g === 'Gender') {
+      this.showGenderGraph = true;
+    } else if (g === 'Age') {
+      this.showAgeGraph = true;
+      this.showMeasurementGenderBins = false;
+    } else if (g === 'Sources') {
+      this.showSourcesGraph = true;
+      this.showMeasurementGenderBins = false;
```

Comment:
nit: why not just set this to false at the start, then set to true under `g === 'Gender'`?